README.md

```diff
@@ -1,6 +1,6 @@
 # Docker ELK stack
 
-[![Join the chat at https://gitter.im/deviantony/docker-elk](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/deviantony/docker-elk?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Elastic Stack version](https://img.shields.io/badge/ELK-5.6.1-blue.svg?style=flat)](https://github.com/deviantony/docker-elk/issues/173)
+[![Join the chat at https://gitter.im/deviantony/docker-elk](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/deviantony/docker-elk?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Elastic Stack version](https://img.shields.io/badge/ELK-5.6.2-blue.svg?style=flat)](https://github.com/deviantony/docker-elk/issues/173)
```

Comment:
You probably want to reference issue 175 here (end of the link) so it points to the changelog.

Reply:
Done!